### ISSUE TITLE
make blending additive default for images?

### ISSUE THREAD

**Issue by tlambert03** [napari/napari#3353]
## 🚀 Feature
Curious whether there's been prior discussion on the default blending mode for images?  Is there good reasoning behind the default of `translucent`?  i find myself always changing it to additive, but perhaps that's too "fluorescence-focused"?

**Comment by sofroniewn** [comment]
i'm open - i could go either way. i don't recall serious discussion, more like whatever "came first" so to speak. Curious what others think

**Comment by chili-chiu** [comment]
Sean McKinney from Stowers microscopy center wants the default to be additive too.

**Comment by jni** [comment]
I'm not 100% on this, and think that maybe it should be a preference (but cue the ever-recurrent warning about not making every decision a preference). I think it very much could be that additive mode is a fluorescence-centric approach. Just a few examples of situations where additive makes no sense:

- any grayscale image (and I'll note that grayscale is our default colormap), including natural images and EM.
- rgb images
- any processing steps, e.g. image -> gaussian blur

Depending on whether you are a fluorescence microscopist, the above use cases might be a tiny minority of your usage or *all* of your usage of napari.

Another idea (possibly not a very good one) is to have default blending be tied to the colormap, so if you create a layer with any of the RGBCMY colormaps, you get additive blending, but any other colormap gets translucent?

**Comment by psobolewskiPhD** [comment]
How is the default set and implemented? it seems it already is doing something "smart":
Loading a single channel tiff I get translucent, but unpacking a 2 channel image via napari-aicsimageio I get additive. I looked through the plugin code and `blending` is never specified, the returns are layer tuples.

Not to belabor this issue, but this switch in behavior leads to issues with Light mode:
https://github.com/napari/napari/issues/3374
(more on this here: https://github.com/napari/napari/discussions/3914)
Personally, I think it's safer to ensure the user sees something and then they can tweak the blending to get the visual they want. Seeing nothing a white canvas suggests something wasn't read/loaded right—which is why I originally thought it was the fault of the plugin and reported there!

Perhaps the default could be set in the preferences?

**Comment by psobolewskiPhD** [comment]
Small update: 
With https://github.com/napari/napari/pull/5487 merged, blending with the canvas is now prevented for additive and minimum blending, instead custom overrides are implemented. 
What this means is that:
- for a single channel the difference between translucent and additive is only apparent if you use Light theme. If you adjust opacity, you can see the difference: translucent does blend to the canvas, while additive will use black so that it works properly. (In Dark mode, there is no functional difference between additive and translucent for single channel images.) This is the case for both RGB layers and grayscale, etc.
- For splitting multichannel and our sample images, we currently use translucent for the base and additive for the rest—this was implemented to guard against blending with canvas. This is slightly annoying if you choose to reorder the layers, but no longer needed because of #5487 overriding the bottom visible layer blending mode.

So basically now it should really just be a question of preference and maybe consistency?
I think it could be worth to go through and drop the `translucent + additive` pattern for split channel and samples to pure additive...
cc @jni
